src/controls/NumericControl.tsx

```diff
@@ -0,0 +1,105 @@
+import { ControlProps } from "@jsonforms/core"
+import { Col, Form, InputNumber, Row, Slider } from "antd"
+import { decimalToPercentage, percentageStringToDecimal } from "./utils"
+
+
+
+export const createNumericControl = (args: { coerceNumber: (value: number) => number; pattern?: string }) => {
+  return function NumericControl({
+    data,
+    handleChange,
+    path,
+    label,
+    visible,
+    id,
+    errors,
+    schema,
+    required,
+  }: ControlProps) {
+    const isValid = errors.length === 0
+    let step = 0
+    const { maximum, minimum } = schema
+    const isRangeDefined = typeof maximum === "number" && typeof minimum === "number"
+    const numberData = data as number | undefined | null
```

Comment:
I added this cast, because the registry was complaining about [NumericControlProps](https://github.com/greatexpectationslabs/mercury-web/blob/b2e7c056451996ceb0933acbbabf36aa866ffad2/src/jsonforms/controls/NumericControl.tsx#L6) as defined in mercury-web.